src/org/loklak/api/amazon/AmazonProductService.java

```diff
@@ -0,0 +1,160 @@
+/**
+ *  AmazonProductService
+ *  Copyright 05.08.2016 by Shiven Mian, @shivenmian
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *  
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program in the file lgpl21.txt
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.loklak.api.amazon;
+
+import java.io.StringWriter;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.json.JSONObject;
+import org.json.XML;
+import org.loklak.server.APIException;
+import org.loklak.server.APIHandler;
+import org.loklak.server.AbstractAPIHandler;
+import org.loklak.server.Authorization;
+import org.loklak.server.BaseUserRole;
+import org.loklak.server.Query;
+import org.loklak.tools.storage.JSONObjectWithDefault;
+import org.w3c.dom.Document;
+
+public class AmazonProductService extends AbstractAPIHandler implements APIHandler {
+
+	private static final long serialVersionUID = 2279773523424505716L;
+
+	// keys available on request
+	private static final String AWS_ACCESS_KEY_ID = "";
+	private static final String AWS_SECRET_KEY = "";
+	private static final String ASSOCIATE_TAG = "";
+
```

Comment:
```java
	private static final String AWS_ACCESS_KEY_ID = "";
	private static final String AWS_SECRET_KEY = "";
	private static final String ASSOCIATE_TAG = "";
```

These are configurable entries, can you please move them to the `conf` file and use the string fields from there ? We wouldn't expect a user to open a java file and make changes to the KeyID and the Key / Tags